liquibase-standard/src/main/java/liquibase/database/LiquibaseTableNamesFactory.java

```diff
@@ -28,6 +28,7 @@ public List<String> getLiquibaseTableNames(Database database) {
     public void destroy(Database abstractJdbcDatabase) throws DatabaseException {
         for (LiquibaseTableNames generator : generators) {
             generator.destroy(abstractJdbcDatabase);
+            abstractJdbcDatabase.commit();
```

Comment:
@filipelautert How do you feel about this change? It was necessary to fix a problem I was seeing where the transaction to destroy the history table was abandoned and never committed, but I also know that we fixed an issue recently where we had too many commits being sent and I don't want to add to it if it's the wrong solution.

Reply:
Interestingly, it seems to be a problem isolated only to Postgres, but I don't know why. I seem to remember reading years ago that Postgres does something different regarding commits, or autocommit, or something like that. But I can't find it and I can't remember it.

Reply:
Correct me if I'm wrong, but this is only called during `dropDatabaseObjects `and it's outside of a loop - in this case is fine.
Previous problem happened because we were doing a commit for every changeset in the changelog file that we read.

And maybe you problem may be related to connection flush or something like that? But anyway 1 commit at the end seems fine .